agent/app/agent_compatibility_unspecified.go

```diff
@@ -20,3 +20,7 @@ import "github.com/aws/amazon-ecs-agent/agent/engine"
 func (agent *ecsAgent) checkCompatibility(engine engine.TaskEngine) error {
 	return nil
 }
+
+func (agent *ecsAgent) scanPlugins() ([]string, error) {
```

Comment:
Doesn't the `Scan()` work on windows?

Reply:
https://docs.docker.com/engine/extend/plugin_api/#plugin-discovery they are specifically scanning for UNIX domain sockets, which is why I've put it under _unix only. Not entirely sure how it works on Windows, I couldn't find any answers googling.

Reply:
https://github.com/moby/moby/pull/25903 I think this solves part of the problem where spec file is supported for now.